test/relative-time.js

```diff
@@ -31,14 +31,14 @@ suite('relative-time', function() {
     const now = new Date(Date.now() + 3 * 60 * 60 * 1000).toISOString()
     const time = document.createElement('relative-time')
     time.setAttribute('datetime', now)
-    assert.equal(time.textContent, '3 hours from now')
+    assert.equal(time.textContent, 'in 3 days')
```

Comment:
Is the change in units from hours to days expected?

Reply:
good catch. I don't think so. let me take a look why this happened.

Reply:
Yep, bug

https://github.com/github/time-elements/blob/2f3e55029379817c7887b34755712f5b253f130d/src/relative-time.js#L136-L139